jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/JibPluginConfiguration.java

```diff
@@ -166,9 +166,10 @@ public void set(String image) {
   @Parameter(defaultValue = "false", required = true)
   private boolean allowInsecureRegistries;
 
+  // this parameter is cloned in FilesMojo
   @Nullable
   @Parameter(defaultValue = "${project.basedir}/src/main/jib", required = true)
-  private String extraDirectory;
+  private File extraDirectory;
```

Comment:
I feel like we used to do this and I had to change it to a String to get rid of a warning, is that not the case anymore?

Reply:
oh, was it some errorprone or nullaway warning? 
Let me make sure this still works even if the directory doesn't exist.

Reply:
Oh actually it might have been a Gradle warning, not Maven.